src/components/Comments/Comments.jsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from "react";
+import { getArticleComments } from "../../utils/util-api-calls";
+import { formatDate } from "../../utils/util-data-formatters";
+import { HiArrowsUpDown, HiMiniUserCircle, HiCalendarDays, HiChevronDown, HiChevronUp, HiChatBubbleLeftEllipsis } from "react-icons/hi2";
+import "./Comments.css";
+
+function Comments({ article_id }) {
+	const [comments, setComments] = useState([]);
+	const [commentsLoading, setCommentsLoading] = useState(true);
+  const [commentsVisible, setCommentsVisible] = useState(true);
+
+	useEffect(() => {
+		setCommentsLoading(true);
+		getArticleComments(article_id)
+			.then((res) => {
+				setCommentsLoading(false);
+				setComments(res.data.comments);
+			})
+			.catch((error) => {
+				console.log(error);
+			});
+	}, []);
+
+  useEffect(()=>{setCommentsVisible(false)}, [])
+
+	return commentsLoading ? (
+		<h2>Loading ...</h2>
+	) : (
+		<section className="comments">
+			<div className="comments-heading" onClick={() => {setCommentsVisible(current => !current)}}>
+        {commentsVisible ? <HiChevronUp className="comments-heading-icon"/> : <HiChevronDown className="comments-heading-icon"/>}
```

Comment:
I really like the icons you have used here, this really enhances your apps usability 😄